src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="link__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">APPLE</a></li>
+
+          <li class="nav__item"><a href="#" class="nav__link">SAMSUNG</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">APPLE</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">SAMSUNG</a>
          </li>
```

Now it's looks cleaner